bindings/tflite/java/build.gradle

```diff
@@ -48,9 +47,12 @@ android {
 
     sourceSets {
         main {
-            java.srcDirs = ['java/org/tensorflow/lite']
-            jni.srcDirs = ['java/org/tensorflow/lite/native']
-            manifest.srcFile 'org/tensorflow/lite/tests/TestManifest.xml'
+            manifest.srcFile 'org/tensorflow/lite/AndroidManifest.xml'
+            java {
+                srcDirs('org/tensorflow/lite')
+                exclude('tests/**') // Don't build tests into the library
+            }
+            jni.srcDirs = ['org/tensorflow/lite/native']
```

Comment:
Are these actually being included? Your screenshot of the .aar shows shared objects from https://github.com/google/iree/tree/main/iree/tools/android/run_module_app , which we _don't_ want in the archive. I'm actually pretty surprised this configuration is managing to pull sources from that directory.